Add notes about invoking adapter methods in Jinja space (#1500)
fishtown-analytics/docs.getdbt.com: website/docs/reference/dbt-jinja-functions/adapter.md

id: "adapter"
---

> dbt implements database communication using an internal database adapter object. For example, BaseAdapter and SnowflakeAdapter. The Jinja object `adapter` is a wrapper around this internal database adapter object.

Reviewer: ```suggestion
Your database communicates with dbt using an internal database adapter object. For example, BaseAdapter and SnowflakeAdapter. The Jinja object `adapter` is a wrapper around this internal database adapter object.
```

> ## Overview

Reviewer: ```suggestion
```

> dbt implements database communication using an internal database adapter object (e.g. BaseAdapter, SnowflakeAdapter). The Jinja object `adapter` is a wrapper around this internal database adapter object.

Reviewer: ```suggestion
dbt implements database communication using an internal database adapter object. For example, BaseAdapter and SnowflakeAdapter. The Jinja object `adapter` is a wrapper around this internal database adapter object.
```
